scripts/rollup.config.js

```diff
@@ -0,0 +1,97 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import pkg from '../package.json';
+
+const date = new Date();
+
+const banner =
+`/**
+ * ${pkg.name}
+ * @version ${pkg.version}
+ * @copyright ${date.getFullYear()} ${pkg.author}
+ * @license ${pkg.license}
+ */`;
+
+export default [
+  /**
+   * Rollup configuration for packaging the plugin in a module that is consumable
+   * as the `src` of a `script` tag or via AMD or similar client-side loading.
+   *
+   * This module DOES include its dependencies.
+   */
+  {
+    name: 'mpdParser',
+    input: 'src/index.js',
+    output: {
+      file: 'dist/mpd-parser.js',
+      format: 'umd'
+    },
+    legacy: true,
+    banner,
+    plugins: [
+      resolve({ browser: true, main: true, jsnext: true }),
+      json(),
+      commonjs({ sourceMap: false }),
+      babel({
```

Comment:
It could make sense to re-use the babel config, I have been meaning to make a pr with it but since they are in the same file it will be easier.

Reply:
Yeah it would be better to use a `.babelrc` so all the builds, including the karma tests, have the same settings and to reduce repetition.